src/VirtoCommerce.MarketingModule.Core/Model/DynamicContent/DynamicContentItem.cs

```diff
@@ -8,6 +8,7 @@ namespace VirtoCommerce.MarketingModule.Core.Model
     public class DynamicContentItem : DynamicContentListEntry, IsHasFolder, IHasDynamicProperties
     {
         public string ContentType { get; set; }
+        public int Priority { get; set; }
```

Comment:
Could you comment, where used this member? I can't find such places.
Does it really need?